CMakeLists.txt

```diff
@@ -376,7 +376,7 @@ set(SRC_FILES
   src/include/trees.h
   src/include/utils.h
   src/init.cc
-  src/init_nvtx.cc
+#  src/init_nvtx.cc
```

Comment:
Do you like to keep it commented or remove entirely?

Reply:
I prefer to keep it commented out, so that we purposelessly know it was left out.